### PR TITLE
Add icons for highway features

### PR DESCRIPTION
**Pull request by Yahia1018**
I met a little problem with the traffic_signals icon. I couldn't find what's wrong with what I did.

Some icons shouldn't have text or halo display, so I choosed the color "rgba(0, 0, 0, 0)" for the elevator for example, is it ok or we could maybe choose this color for the fallback value of the text-color ?

### CODE REVIEW

**Comment by bchapuis** [comment]
The icon is named traffic_light, so there is a mismatch between the icon name and the attribute name in OpenStreetMap. There are probably a few other exceptions.

https://github.com/baremaps/openstreetmap-icons/blob/main/icons/traffic_light.svg

Could you create an issue and list these exceptions as you encounter them? I'm not sure about the best approach to adopt  yet (renaming the icon, handling the exception with a special case in the style).

**Comment by Yahia1018** [comment]
I see, thank you. Yes, I create this right now.

File: style.json [hunk 1]
```diff
@@ -861,14 +861,14 @@
...
...entre", "casino", "cinema", "community_centre", "nightclub", "public_bookcase", "theatre", "courthouse", "firestation", "police", "post_box", "post_office", "prison", "townhall", "bbq", "bench", "drinking_water", "shelter", "shower", "telephone", "toilets", "recycling", "waste_basket", "waste_disposal", "internet_cafe", "public_bath" ] ] ], "rgba(115, 74, 10, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "kick-scooter_rental", "bicycle_parking", "bicycle_rental", "boat_rental", "bus_station", "car_rental", "charging_station", "fuel", "motorcycle_parking", "parking", "parking_entrance", "taxi" ] ] ], "rgba(0, 146, 218, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "clinic", "dentist", "doctors", "hospital", "pharmacy", "veterinary" ] ] ], "rgba(191, 0, 0, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "fountain" ] ] ], "rgba(87, 104, 236, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "marketplace" ] ] ], "rgba(172, 57, 172, 1)", [ "has", "historic" ], "rgba(115, 74, 10, 1)", [ "in", [ "get", "highway" ], [ "literal", [ "bus_stop" ] ] ], "rgba(86, 148, 252, 1)", [ "in", [ "get", "highway" ], [ "literal", [ "elevator" ] ] ], "rgba(0, 0, 0, 0)", "rgba(199, 116, 0, 1)" ],
```
Comment (bchapuis) [inline style.json:864]: The cases for the text color and icon colors should be the same, right?

File: style.json [hunk 2]
```diff
@@ -861,14 +861,14 @@
...
       "icon-opacity" : 1,
       "icon-translate-anchor" : "map",
       "icon-halo-color" : "rgba(255, 255, 255, 0.8)",
       "icon-halo-width" : 1,
       "text-halo-width" : 1,
-      "text-halo-color" : "rgba(255, 255, 255, 0.8)"
+      "text-halo-color" : [ "case", [ "in", [ "get", "highway" ], [ "literal", [ "elevator" ] ] ], "rgba(0, 0, 0, 0)", "rgba(255, 255, 255, 0.8)" ]
```
Comment (bchapuis) [inline style.json:871]: Shouldn't we have a white halo for the elevator?

File: style.json (hunk 1 above)
Comment (Yahia1018) [reply style.json:864]: Generally yes, but bus_stop has a different text color from icon color.

Comment (Yahia1018) [reply style.json:864]: Maybe I didn't see well and it is the halo who makes a little difference for the eyes. I changed and put the same color for both.

**Review by bchapuis — APPROVED** [review]

**Pull request merged**